src/integrations/HttpIntegration.ts

```diff
@@ -252,11 +253,20 @@ class HttpIntegration implements Integration {
                             res.on('end', () => {
                                 try {
                                     if (chunks && chunks.length) {
-                                        const responseBody: string = Buffer.concat(chunks).toString('utf8');
+                                        const concatedChunks = Buffer.concat(chunks);
+                                        const contentEncoding = HTTPUtils.obtainIncomingMessageEncoding(res);
+                                        const responseBody: string =
+                                            contentEncoding
+                                                ? (EncodingUtils.getPayload(
```

Comment:
What if Content-Encoding have [multiple compression algorithm](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding)?

e.g: 

// Multiple, in the order in which they were applied
Content-Encoding: deflate, gzip